src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+const listItems = [...tree.querySelectorAll('li')];
+
+listItems.forEach((elem) => {
+  const firstText = elem.firstChild;
+  const span = document.createElement('span');
+
+  span.textContent = `${firstText.textContent}`;
+
+  elem.replaceChild(span, firstText);
+});
+
+// eslint-disable-next-line no-shadow
+function clickOnSpan(event) {
+  const spanTarget = event.target.closest('span');
+
+  if (spanTarget) {
+    const list = spanTarget.nextElementSibling;
```

Comment:
The code assumes that `spanTarget.nextElementSibling` is a valid element. If there is no sibling element, `list` will be `null`, and calling `getComputedStyle` on it will result in an error. Consider adding a check to ensure `list` is not `null` before proceeding.